{
	"vid": "67fe5e5b-549f-4a2a-9834-53f60eaa415e",
	"topic": "py-Scrapy -- exposure of sensitive information vulnerability",
	"affects": [
		{
			"name": [
				"py37-Scrapy",
				"py38-Scrapy",
				"py39-Scrapy",
				"py310-Scrapy",
				"py311-Scrapy"
			],
			"range": [
				{
					"lt": "2.6.1"
				}
			]
		}
	],
	"description": {
		"xmlns": "http://www.w3.org/1999/xhtml",
		"text": "\n    <p>ranjit-git reports:</p>\n    <blockquote cite=\"https://osv.dev/vulnerability/PYSEC-2022-159\">\n      <p>Exposure of Sensitive Information to an Unauthorized Actor in GitHub repository scrapy/scrapy prior to 2.6.1.</p>\n    </blockquote>\n      "
	},
	"references": {
		"url": [
			"https://osv.dev/vulnerability/PYSEC-2022-159",
			"https://osv.dev/vulnerability/GHSA-cjvr-mfj7-j4j8"
		],
		"cvename": [
			"CVE-2022-0577"
		]
	},
	"dates": {
		"discovery": "2022-03-02",
		"entry": "2023-08-31"
	}
}
